app/Models/Category.php

```diff
@@ -6,14 +6,22 @@
 
 class Category extends Model
 {
+    protected $hidden = [
+        'parent_id',
+        'level',
+        'created_at',
+        'updated_at',
+        'pivot',
```

Comment:
sao lại có trường pivot ở đây? nó có ý nghĩa gì v?

Reply:
Để nó bỏ cái trường `pivot` trong `json response` ấy chị. `Product` với `Category` là many-to-many nên khi truy vấn `Product::with('categories')->get();` có trường `pivot` chứa `product_id` và `category_id`.